server/server_test.go

```diff
@@ -575,6 +575,63 @@ func (cli *testServerClient) runTestLoadDataAutoRandom(c *C) {
 	})
 }
 
+func (cli *testServerClient) runTestLoadDataAutoRandomWithSpecialTerm(c *C) {
+	path := "/tmp/load_data_txn_error_term.csv"
```

Comment:
I think it's better to convert this test to an integration test. BTW, the `/tmp/load_data_txn_error_term.csv` file is not exists on the repo